cmd/experimental/kueue-viz/frontend/Dockerfile

```diff
@@ -1,5 +1,5 @@
 #Build the React app
-FROM node:23 AS build
+FROM quay.io/jitesoft/node:23 AS build
```

Comment:
Why this change?

We probably have a redhat registry item for this?

Reply:
The previous registry hit the Docker rate-limit issue, so I was looking for a well-maintained Quay registry.

Reply:
We should use the Red Hat provided nodejs repos: https://catalog.redhat.com/software/containers/rhel9/nodejs-22/66431c955b4f8d8ffc017905